CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## 0.2.0 - 2020-08-03
+
+### Added
```

Comment:
Is it worth mentioning the addition of `proxy_stream_type_t`, or not really since that is just supplementary to added functions?

Reply:
Good question, but I don't this so. It doesn't exist on its own, and we'd be adding noise into change log.

Reply:
Yeah, makes sense.